pkgs/tools/misc/rlwrap/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "rlwrap";
-  version = "0.46";
+  version = "0.46.1";
 
   src = fetchFromGitHub {
     owner = "hanslub42";
     repo = "rlwrap";
     rev = "v${version}";
```

Comment:
Does this work? Noticed that the 0.46.1 tag doesn't have the `v` prefix:
https://github.com/hanslub42/rlwrap/tags